javascript-arrays/main.js

```diff
@@ -0,0 +1,24 @@
+var colors;
+colors = ['red', 'white', 'blue'];
+
+console.log('value of colors[0]:', colors[0]);
+console.log('value of colors[1]:', colors[1]);
+console.log('value of colors[2]:', colors[2]);
+
+console.log('America is', colors[0] + ', ' + colors[1] + ', and' + ' ' + colors[2] + '.');
+
+colors[2] = 'green';
+console.log('Mexico is', colors[0] + ', ' + colors[1] + ', and' + ' ' + colors[2] + '.');
+
+console.log(colors);
+
+var students = ['Jane', 'Joe', 'John', 'Sally'];
+var numberOfStudents;
+numberOfStudents = students.length;
```

Comment:
You can do this all on one line, there's no need to separate this to multiple lines.